src/main/java/nextstep/subway/domain/Sections.java

```diff
@@ -0,0 +1,75 @@
+package nextstep.subway.domain;
+
+import nextstep.subway.applicaion.dto.StationResponse;
+import nextstep.subway.exception.ValidationException;
+
+import javax.persistence.*;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.stream.Collectors;
+import java.util.stream.Stream;
+
+@Embeddable
+public class Sections {
+    @OneToMany(mappedBy = "line",
+            cascade = {CascadeType.PERSIST, CascadeType.MERGE},
+            orphanRemoval = true)
+    private List<Section> sections = new ArrayList<>();
+    private int DELETION_MINIMUM_SIZE = 1;
```

Comment:
상수 정의 👍 

```suggestion
    private static final int DELETION_MINIMUM_SIZE = 1;
```
static final 로 정의하는 걸 추천드립니다

또한 상수는 인스턴스 변수 위에 정의하는 게 일반적인 컨벤션 입니다 😅